Movie/Movie/Helper/Constants.swift

```diff
@@ -45,12 +45,30 @@ enum Constants {
     static let APIKeyName = "APIKey"
     static let APIKeyValue = "300c9789c349380a810d91b042610da7"
```

Comment:
```suggestion
    static let APIKeyValue = ""
```
этого быть не должно